backend/postgres/postgres.go

```diff
@@ -271,10 +271,11 @@ func openTracingAdapter(event *pg.QueryProcessedEvent) {
 		q = qe.Error()
 	}
 
-	name := fmt.Sprintf("PostgreSQL: %s", q)
-	span, _ := opentracing.StartSpanFromContext(event.DB.Context(), name,
+	span, _ := opentracing.StartSpanFromContext(event.DB.Context(), "sql:query",
```

Comment:
I would add the first term e.g. INSERT, SELECT, ... to he query as well. e.g. `sql:insert`